code/datums/gamemodes/game_mode.dm

```diff
@@ -21,6 +22,12 @@
 	var/on_distress_cooldown = FALSE
 	var/waiting_for_candidates = FALSE
 
+	// Xeno round start conditions
+	var/xeno_required_num = 1 // Number of xenos required to start 
+	var/xeno_starting_num = 0 // Number of xenos given at start
```

Comment:
It's the number xenos started with, initializing it to 0 makes no sense and it can be misleading as there's a difference between init with 0 and call the proc and make it 0